packages/riipen-ui/src/components/List.test.jsx

```diff
@@ -0,0 +1,61 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import List from "./List";
+
+describe("<List>", () => {
+  it("renders without errors", () => {
+    let error;
+
+    try {
+      mount(<List />);
+    } catch (theError) {
+      error = theError;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  describe("snapshot test", () => {
+    it("renders correctly", () => {
+      const wrapper = mount(<List />);
+
+      expect(toJson(wrapper)).toMatchSnapshot();
+    });
+  });
```

Comment:
@imogenem @licarm another example of the snapshot test. Let's update this to be whatever we choose.